qcodes/instrument_drivers/basel/sp983c_remote.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any, Dict, Optional
+import numpy as np
+
+from qcodes.instrument.visa import VisaInstrument
+from qcodes.instrument.parameter import Parameter, DelegateParameter
+from qcodes.utils import validators as vals
+
+
+class SP983A(VisaInstrument):
+    """
+    A driver for Basel Preamp's (SP983C) Remote Instrument - Model SP983A.
```

Comment:
since we have a notebook for the other basel driver, we probably should mention the remote one there as well --> https://qcodes.github.io/Qcodes/examples/driver_examples/Qcodes%20example%20with%20Basel%20SP983c%20Preamp.html?highlight=basel

Reply:
added remote's information to the notebook